404.md

```diff
@@ -6,6 +6,14 @@ notoc: true
 tree: false
 ---
 
+<br/>
+
+We tried to forward you to where we think you might be going, but we couldn't
+find a good match. You've got some choices:
+
+<div id="sorryMsg" />
```

Comment:
Better to change this to

```html
<div id="sorryMsg"></div>
```

http://stackoverflow.com/questions/3558119/are-non-void-self-closing-tags-valid-in-html5